iphone/Classes/TiUIListView.m

```diff
@@ -55,18 +55,18 @@ @implementation TiUIListView {
     CGPoint tapPoint;
     BOOL editing;
     BOOL pruneSections;
-    
```

Comment:
Same thing with the blank lines. Probably you edited the file in Atom instead of Xcode? Try the project in `titanium_mobile/iphone/iphone/Titanium.xcodeproj` and edit the app.js in `titanium_mobile/iphone/Resources` to run it from Xcode directly.